.eslintrc.js

```diff
@@ -13,8 +13,11 @@ module.exports = {
   ],
   overrides: [
     {
-      files: ['*.test.{ts,js}'],
+      files: ['*.test.{ts,tsx,js}'],
       extends: ['@metamask/eslint-config-jest'],
+      rules: {
+        '@typescript-eslint/no-shadow': ['error', { allow: ['Text'] }],
```

Comment:
Prevents this error

<img width="517" alt="Screenshot 2024-10-31 at 2 59 17 PM" src="https://github.com/user-attachments/assets/41b443bc-b97b-4b12-bc8f-caaf36c4bd21">

`Text` is frequently used as a component name, especially in UI libraries like React Native or styled-components. Allowing Text helps prevent unnecessary linting errors when importing or defining UI components.